sdk/storage/azure-storage-file-share/azure/storage/fileshare/_lease.py

```diff
@@ -61,25 +68,33 @@ def __exit__(self, *args):
         self.release()
 
     @distributed_trace
-    def acquire(self, **kwargs):
+    def acquire(self, lease_duration=-1, **kwargs):
```

Comment:
I'm good with this! Since it's not a breaking change. While I still feel it would be safer to ask for Anna's opinion @annatisch are you good with this?

Reply:
Could this not just be rolled into kwargs?
I don't really see a reason it needs to be in method signature...

Reply:
This would be inconsistent to `\blob\_lease.py`, do we not want to maintain the same method definition/pattern as `\blob\_lease.py`?

Reply:
It's not really inconsistent with Blobs, because the parameter is the same. But if this parameter only applies to Share leases, we should not make it so prominent in the method signature. I think kwargs would be best :)